lib/i18n/app_de.json

```diff
@@ -2,6 +2,13 @@
   "common": {
     "appName": "Grüne App"
   },
+  "bottomNavigation": {
+    "news": "Artikel",
+    "campaigns": "Wahlkampf",
+    "profiles": "Mitglieder",
+    "mfa": "2FA",
+    "apps": "Tools"
+  },
```

Comment:
:wrench: I think we should avoid implementation-dependent namespaces like this. Instead, these translations should go to the corresponding feature namespace. You can reuse the ones I already created and just fix the naming.
Example: `news.news` instead of `bottomNavigation.news`.

Reasoning:
- If we ever change the navigation method and remove bottom navigation bar, all keys have to be changed
- If these keys are ever needed in another place, we have to duplicate them (or use `bottomNavigation.xyz` in e.g. the settings, even if it has nothing to do with `bottomNavigation`
- Consistency with the folder/route/project structure

Reply:
I think it's good to have the translations for the bottom navigation bar in its own scope.
For example, in the bottom navigation bar it should be "2FA", in the page title "Zwei-Faktor-Authentifizierung".